src/status_im/contexts/wallet/sheets/network_selection/view.cljs

```diff
@@ -0,0 +1,79 @@
+  (let [{network-name :network-name
+         chain-id     :chain-id}                 network
+        {balances-per-chain :balances-per-chain} token
+        balance-for-chain                        (get balances-per-chain chain-id)
+        total-balance                            (money/bignumber (:balance balance-for-chain))
+        fiat-value                               (utils/calculate-token-fiat-value
+                                                  {:currency currency
+                                                   :balance  total-balance
+                                                   :token    token})
+        crypto-formatted                         (utils/get-standard-crypto-format token total-balance)
+        fiat-formatted                           (utils/get-standard-fiat-format crypto-formatted
+                                                                                 currency-symbol
+                                                                                 fiat-value)
+        token-symbol                             (:symbol token)
+        mainnet?                                 (= network-name :mainnet)]
+    [rn/view {:style (style/network-list-container mainnet?)}
+     [quo/network-list
+      {:label         (name network-name)
+       :network-image (quo.resources/get-network network-name)
+       :token-value   (str crypto-formatted " " token-symbol)
+       :fiat-value    fiat-formatted
+       :on-press      #(on-select-network network)}]]))
+
+(defn view
+  [{:keys [token on-select-network]}]
+  (let [{token-networks :networks
+         token-symbol   :symbol} token
+        mainnet-network          (first
+                                  (filter
+                                   (fn [network]
+                                     (= (:network-name network) :mainnet))
+                                   token-networks))
+        layer-2-networks         (filter
+                                  (fn [network]
+                                    (not= (:network-name network) :mainnet))
+                                  token-networks)
+        currency                 (rf/sub [:profile/currency])
+        currency-symbol          (rf/sub [:profile/currency-symbol])]
+    [:<>
+     [rn/view {:style style/header-container}
+      [quo/text
+       {:size   :heading-2
+        :weight :semi-bold}
+       (i18n/label :t/select-network)]
+      [quo/context-tag
+       {:type            :token
+        :size            24
+        :token           token-symbol
+        :container-style style/context-tag}]]
+     (when mainnet-network
+       [network-item
+        {:network           mainnet-network
+         :token             token
+         :on-select-network on-select-network
+         :currency          currency
+         :currency-symbol   currency-symbol}])
+     [quo/divider-label {:container-style style/divider-label}
+      (i18n/label :t/layer-2)]
+     [rn/flat-list
```

Comment:
https://github.com/status-im/status-mobile/assets/19279756/750e00ef-6d89-411b-ac1e-0ebe98b12246

Is this scrolling behavior expected? If not, we can use `section-list` instead of `flat-list` to avoid it.

Reply:
Sometime we don't need to show Mainnet, so section list won't be needed. Maybe just disable scrolling? wdyt?

Reply:
Disabling the scroll should be enough, thats a nice idea 🤝